lib/graph/Collaboratory/Course.js

```diff
@@ -0,0 +1,40 @@
+// dependencies
+var Vertex = require("../Vertex");
+var utils = require("../../utils");
+
+
+// single export
+module.exports = COL_Course;
+
+
+/**
+ * Represents a course (group) vertex
+ *
+ * @constructor
+ * @param {Client} client  The API client object (passed around internally)
+ * @param {String} id      UUID
+ */
+function COL_Course(client, id) {
+    Vertex.call(this, client, id);
+}
+
+
+// inheritance
+Vertex.extend(COL_Course);
+
+
+// override for Vertex#base
+COL_Course.prototype.base = "col/courses";
+
+
```

Comment:
We need a method to get a single course by ID. viz., "COL_Course.prototype.get()"

Reply:
We get it from the inherence of Vertex.  So the syntax is: `NH.api.collab().course(id).get(function(err, course) {...}`

Reply:
disregard! thanks for pointing out that this is already available.